docs/src/Table.doc.js

```diff
@@ -341,193 +348,211 @@ card(
   <Example
     name="Example: Table Row Expandable"
     defaultCode={`
-<Table>
-  <Table.Header>
-    <Table.Row>
-      <Table.HeaderCell>
-        <Box display="visuallyHidden">
-          <Text weight="bold">Open/Close row</Text>
-        </Box>
-      </Table.HeaderCell>
-      <Table.HeaderCell>
-        <Text weight="bold">Name</Text>
-      </Table.HeaderCell>
-      <Table.HeaderCell>
-        <Text weight="bold">House</Text>
-      </Table.HeaderCell>
-      <Table.HeaderCell>
-        <Text weight="bold">Birthday</Text>
-      </Table.HeaderCell>
-    </Table.Row>
-  </Table.Header>
-  <Table.Body>
-    <Table.RowExpandable
-      accessibilityExpandLabel="Expand"
-      accessibilityCollapseLabel="Collapse"
-      id="row1"
-      expandedContents={
-        <Box maxWidth={236} padding={2} column={12}>
-          <Card
-            image={
-              <Avatar
-                name="luna avatar"
-                src="https://i.ibb.co/QY9qR7h/luna.png"
-              />
-            }
-          >
-            <Text align="center" weight="bold">
-              <Link href="https://pinterest.com">
-                <Box paddingX={3} paddingY={2}>
-                  Luna's Info
-                </Box>
-              </Link>
-            </Text>
-          </Card>
-        </Box>
+    function RowExpandableExample() {
+      const [iconShown, setIconShown] = React.useState(false);
```

Comment:
@faisalgedi Instead of showing an icon, could we make it more explicit and toggle some text? E.g.

`<Text>Row expanded</Text>` 

![image](https://user-images.githubusercontent.com/127199/96279139-2628a200-0f8b-11eb-93ca-60bfc52b5685.png)